1.1/translations/validator_de.ttl

```diff
@@ -0,0 +1,877 @@
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix geo: <http://www.opengis.net/ont/geosparql#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix sdo: <https://schema.org/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@base <http://www.opengis.net/def/geosparql/validator/> .
+
+
+<http://www.opengis.net/def/geosparql/validator>
+	a owl:Ontology ;
+  	dcterms:title "GeoSPARQL RDF Shapes Validator"@de ;
+	dcterms:description """Diese [Shapes Constraint Language (SHACL)](https://www.w3.org/TR/shacl/) Datei definiert Beschränkungen für RDF Daten. Es kann zur Verifizierung der Konformität zum GeoSPARQL Standard verwendet werden.
+	
+Für GeoSPARQL 1.1, ist dieser Validator nicht normativ, sondern nur informativ, er wird aber wahrscheinlich die Grundlage für zukünftige, normative Validatoren sein."""@de ;
+	dcterms:publisher [
+		a sdo:Organization ;
+		sdo:name "Open Geospatial Consortium" ;
+		sdo:url "https://www.ogc.org"^^xsd:anyURI ;
+	] ;
+	dcterms:creator "OGC GeoSPARQL Standards Working Group" ;
+	dcterms:created "2021-06-13"^^xsd:date ;
+	dcterms:modified "2021-08-12"^^xsd:date ;
+    dcterms:license "https://www.ogc.org/license"^^xsd:anyURI ;
+	dcterms:rights "(c) 2021 Open Geospatial Consortium" ; 	
+	owl:versionInfo "OGC GeoSPARQL 1.1" ;
+	owl:versionIRI <http://www.opengis.net/def/geosparql/validator/1.1> ;
+.
+
+#################################################
+# Shape 1 (1a-1e)
+#################################################
+
+<S1-hasGeometry-hasSerialization>
+    a sh:NodeShape ;
+	sh:targetObjectsOf geo:hasGeometry ;
+	sh:property <S1-a-hasGeometry-hasSerialization-sub> , <S1-b-hasGeometry-hasSerialization-sub> , <S1-b-hasGeometry-hasSerialization-sub> , <S1-c-hasGeometry-hasSerialization-sub> , <S1-d-hasGeometry-hasSerialization-sub> , <S1-e-hasGeometry-hasSerialization-sub> ;
```

Comment:
Please note that the `sh:property` object of `<S1-b-hasGeometry-hasSerialization-sub>` is duplicated here, and Line 44 and Line 49. I have de-duplicated these in the master validator.ttl with a direct commit, so these will need de-duplication also.